plugins/Installation/lang/en.json

```diff
@@ -1,14 +1,14 @@
 {
     "Installation": {
-        "CollaborativeProject": "Matomo is a collaborative project, built with love by people from all over the world.",
-        "ConfigurationHelp": "Your Matomo configuration file appears to be misconfigured. You can either remove config\/config.ini.php and resume installation, or correct the database connection settings.",
-        "ConfirmDeleteExistingTables": "Are you sure you want to delete the tables: %s from your database? WARNING: DATA FROM THESE TABLES CANNOT BE RECOVERED!",
+        "CollaborativeProject": "Matomo is a collaborative project built with love.",
+        "ConfigurationHelp": "Correct your Matomo configuration file by either either removing config\/config.ini.php and resuming installation, or correct the database connection settings.",
+        "ConfirmDeleteExistingTables": "Delete the %s tables from your database? WARNING: DATA FROM THESE TABLES CANNOT BE RECOVERED!",
         "Congratulations": "Congratulations",
-        "CongratulationsHelp": "<p>Congratulations! Your Matomo installation is complete.<\/p><p>Make sure your JavaScript code is entered on your pages, and wait for your first visitors!<\/p>",
+        "CongratulationsHelp": "<p>Congratulations! Your Matomo installation is complete.<\/p><p>Ensure your tracking code is entered on your pages, and wait for your first visitors.<\/p>",
         "DatabaseAbilities": "Database abilities",
         "DatabaseCreation": "Database creation",
-        "DatabaseErrorConnect": "Error while trying to connect to the database server",
-        "DatabaseServerVersion": "Database server version",
+        "DatabaseErrorConnect": "Could not connect to the database server",
+        "DatabaseServerVersion": "Database-server version",
```

Comment:
These two are probably better to revert also. "Database-server" doesn't conform to existing language here, and "Error" is maybe more clear (little bit 50/50 on that one)

Reply:
It isn't the server version for the database.
Triple noun needs the dash.
Is the connection made or not? This is what I want to know, not when the error happened.
"while trying" isn't specifying anything. There is no "while not trying".